airflow/contrib/operators/gcp_automl_operator.py

```diff
@@ -0,0 +1,992 @@
+        page_iterator = hook.list_table_specs(
+            dataset_id=self.dataset_id,
+            filter_=self.filter_,
+            page_size=self.page_size,
+            location=self.location,
+            project_id=self.project_id,
+            retry=self.retry,
+            timeout=self.timeout,
+            metadata=self.metadata,
+        )
+        result = [MessageToDict(spec) for spec in page_iterator]
+        self.log.info(result)
+        self.log.info("Table specs obtained.")
+        return result
+
+
+class AutoMLDatasetListOperator(BaseOperator):
+    """
+    Lists table specs in a dataset.
+
+    :param project_id: ID of the Google Cloud project where datasets are located if None then
+        default project_id is used.
+    :type project_id: str
+    :param location: The location of the project.
+    :type location: str
+    :param retry: A retry object used to retry requests. If `None` is specified, requests will not be
+        retried.
+    :type retry: Optional[Retry]
+    :param timeout: The amount of time, in seconds, to wait for the request to complete. Note that if
+        `retry` is specified, the timeout applies to each individual attempt.
+    :type timeout: Optional[float]
+    :param metadata: Additional metadata that is provided to the method.
+    :type metadata: Optional[Sequence[Tuple[str, str]]]
+    :param gcp_conn_id: The connection ID to use to connect to Google Cloud Platform.
+    :type gcp_conn_id: str
+    """
+
+    template_fields = ("location", "project_id")
+
+    @apply_defaults
+    def __init__(
+        self,
+        location: str,
+        project_id: str = None,
+        metadata: Sequence[Tuple[str, str]] = None,
+        timeout: float = None,
+        retry: Retry = None,
+        gcp_conn_id: str = "google_cloud_default",
+        *args,
+        **kwargs
+    ):
+        super().__init__(*args, **kwargs)
+        self.location = location
+        self.project_id = project_id
+        self.metadata = metadata
+        self.timeout = timeout
+        self.retry = retry
+        self.gcp_conn_id = gcp_conn_id
+
+    def execute(self, context):
+        hook = AutoMLHook(gcp_conn_id=self.gcp_conn_id)
+        self.log.info("Requesting datasets")
+        page_iterator = hook.list_datasets(
+            location=self.location,
+            project_id=self.project_id,
+            retry=self.retry,
+            timeout=self.timeout,
+            metadata=self.metadata,
+        )
+        result = [MessageToDict(dataset) for dataset in page_iterator]
+        self.log.info("Datasets obtained.")
+
+        self.xcom_push(
+            context, key="dataset_id_list", value=[get_object_id(d) for d in result]
+        )
+        return result
+
+
+class AutoMLDeleteDatasetOperator(BaseOperator):
+    """
+    Deletes a dataset and all of its contents.
+
+    :param dataset_id: Name of the dataset_id, list of dataset_id or string of dataset_id
```

Comment:
A fragment with a backreference to the guide is missing 
https://github.com/apache/airflow/blob/master/airflow/contrib/operators/gcp_spanner_operator.py#L153-L155

Reply:
Should we add it to Munchkin or is it already present?

Reply:
We should add.